source/Calamari.Tests/KubernetesFixtures/Authentication/SetupKubectlAuthenticationFixture.cs

```diff
@@ -172,8 +186,8 @@ public void Execute_WhereNamespaceDoesNotExist_NamespaceCreated()
 
             invocations.TakeLast(2).Should().BeEquivalentTo(new[]
             {
-                ("kubectl", $"get namespace {Namespace} --request-timeout=1m"),
-                ("kubectl", $"create namespace {Namespace} --request-timeout=1m"),
+                ("kubectl", $"get namespace {Namespace}"),
+                ("kubectl", $"create namespace {Namespace}"),
```

Comment:
Because the test uses a substitute `IKubectl`, but the real `Kubectl` contains the request timeout flag, this arg doesn't show up in any of the test commands anymore